### PR TITLE
MES 4678: Analytics for Cat A Mod 2 Post Test Screens

### PR DESCRIPTION
**Pull request by faggc**
## Description
Adds analytics for office page fields:
+ Mode of transport
+ Independent driving

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by appleabs — COMMENTED** [review]
missing tests

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/pages/office/__tests__/office.analytics.effects.spec.ts [hunk 1]
```diff
@@ -387,6 +392,7 @@ describe('Office Analytics Effects', () => {
       actions$.next(new catAMod1TestSummaryActions.CircuitTypeChanged(CircuitType.Left));
       // ASSERT
       effects.setCircuit$.subscribe((result) => {
+        console.log(result);
```
Comment (appleabs) [inline src/pages/office/__tests__/office.analytics.effects.spec.ts]: console.log left in

**Review by appleabs — APPROVED** [review]
Looks good! Just need to remove the console.log 👍

**Pull request merged**